app/models/event_role.rb

```diff
@@ -115,7 +115,11 @@ def to_s
   def assigned_to(options = {})
     if assigned?
       if options[:use_display_name]
-        member.display_name
+        if options[:use_both_names]
+          "#{member.display_name} (#{member.fullname})"
+        else
+          member.display_name
+        end
```

Comment:
I don't know if this particular nesting is the best option -- IMO `use_both_names` should take precedence when true and not be conditional on `use_display_name`.

Reply:
Changed the function so it doesn't take options so this should not be an issue anymore